Podporovat Flask 3 nebo omezit verzi v závislostech

Naše stránky se nedají vytvořit s Flaskem 3.0.0.
Je třeba buď zjistit, co se rozbilo a přizpůsobit kód novému Flasku, nebo omezit verzi v závislostech.

CI failuje s:

```
Run python pyladies_cz.py freeze --verbose
Traceback (most recent call last):
Generating HTML...
  File "/home/runner/work/pyladies.cz/pyladies.cz/pyladies_cz.py", line 329, in <module>
    cli(app, freezer=freezer, base_url='http://pyladies.cz/')
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/elsa/_cli.py", line 174, in cli
    return command()
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/click/core.py", line 1157, in __call__
    return self.main(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/click/core.py", line 1078, in main
    rv = self.invoke(ctx)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/click/core.py", line 1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/click/core.py", line 1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/click/core.py", line 783, in invoke
    return __callback(*args, **kwargs)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/elsa/_cli.py", line 125, in freeze
    freeze_app(app, freezer, path, base_url, verbose=verbose)
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/elsa/_cli.py", line 56, in freeze_app
    for page in freezer.freeze_yield():
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/flask_frozen/__init__.py", line 183, in freeze_yield
    for url, endpoint, last_modified in self._generate_all_urls():
  File "/opt/hostedtoolcache/Python/3.10.13/x64/lib/python3.10/site-packages/flask_frozen/__init__.py", line 232, in _generate_all_urls
    url_encoding = self.app.url_map.charset
AttributeError: 'Map' object has no attribute 'charset'
Error: Process completed with exit code 1.
```

--- a/requirements.txt
+++ b/requirements.txt
@@ -2,3 +2,4 @@ PyYAML
 elsa>=0.1.2
 markdown
 markupsafe
+werkzeug<3

